# The header of this sample
std_msgs/Header header
# The time this sample was received.
time recv_time
